indico/modules/events/templates/display/indico/meeting.html

```diff
@@ -61,8 +61,17 @@ <h1 itemprop="name">
         {% endif %}
 
         {% if event.has_note %}
-            {{ render_note(event.note, hidden=not theme_settings.show_notes, can_edit=event.can_manage(session.user),
-                           for_event=event) }}
+            {# Check if url contains ?note=note_id and if so, expand the note #}
+            {% set req = request.args.get('note')|string %}
```

Comment:
it's always a string (or none if not present), so no need to convert

Reply:
fixed ;)